### ISSUE TITLE
Deprecation Warning in TF v0.11.13

### ISSUE THREAD

**Issue by JustinGrote** [Azure/terraform-azurerm-loadbalancer#19]
Getting this warning for the public IP, should be updated to use allocation_method

Warning: module.loadbalancer.azurerm_public_ip.azlb: "public_ip_address_allocation": [DEPRECATED] this property has been deprecated in favor of `allocation_method` to better match the api

**Comment by aidanSoles** [comment]
Getting the same error here:
```
$ terraform apply

Warning: module.consul_servers_lb.azurerm_public_ip.azlb: "public_ip_address_allocation": [DEPRECATED] this property has been deprecated in favor of `allocation_method` to better match the api
```

**Comment by yupwei68** [comment]
Hi @JustinGrote , thanks for opening this issue. Sorry for the delay. We have released v2.0.0 to support Terraform v0.12 and have replaced deprecated variable `allocation_method`. And sorry that I have to close this issue.
